python/pyarrow/ipc.pxi

```diff
@@ -1065,7 +1066,7 @@ def read_message(source):
     return result
 
 
-def read_schema(obj, DictionaryMemo dictionary_memo=None):
+def read_schema(obj, DictionaryMemo dictionary_memo=None, use_memory_map=False):
```

Comment:
```suggestion
def read_schema(obj, DictionaryMemo dictionary_memo=None):
```

This is expected to work on `buffer` or `Message`, none of them can be memory_mapped